src/js/edu-benefits/utils/validations.js

```diff
@@ -212,6 +212,17 @@ function isValidSeparatedDateField(date, dateEntered) {
   return true;
 }
 
+function isValidRepayingPeriodToDate(date, dateStarted) {
```

Comment:
The name seems overly specific given that it's just checking to see which date comes first.  Maybe something like `isDateBefore`?

Also, I would encourage the use of momentjs here (and everywhere we have to manipulate dates).